app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,69 @@
+/*
+* This is a manifest file that'll be compiled into application.css, which will include all the files
+* listed below.
+*
+* Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+* or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+*
+* You're free to add application-wide styles to this file and they'll appear at the bottom of the
+* compiled file so the styles you add here take precedence over styles defined in any styles
+* defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+* file per style scope.
+*
+*= require_tree .
+*= require_self
+*/
+
+body {
+  background-image: url('paper.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+  text-align: center;
+}
+
+.new_post {
+  display: inline-block;
+}
+
+.post_input {
+  height: 20%;
+}
+
+.post_title {
+  padding-top: 2%;
+  padding-bottom: .5%;
+  font-family: 'Josefin Slab', sans-serif;
```

Comment:
You could eliminate all this repetition of 'Josefin Slab' by assigning font-family to body...